src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -185,15 +185,11 @@ def reinit_data(self):
 
     def _get_if_entry(self, if_name):
         if_table = ""
-
-        # Once PORT_TABLE will be moved to CONFIG DB
-        # we will get entry from CONFIG_DB for all cases
-        db = mibs.APPL_DB
+        db = mibs.CONFIG_DB
         if if_name in self.if_name_map:
-            if_table = mibs.if_entry_table(if_name)
+            if_table = mibs.if_entry_table_config_db(if_name)
```

Comment:
Why we cannot get everything from APP_DB? As you mentioned, "`oper_status`" is only in APP_DB. Also if there is no config_db entry for `MTU`, then there is a default `9100 `MTU created only in APP_DB table.